### ISSUE TITLE
spurious warning with link-args inside include!

### ISSUE THREAD

**Issue by comex** [rust-lang/rust#19054]
The link args *are* being used, but rustc thinks they're not.

```
% cat a.rs
#![feature(link_args)]
include!("c.rs")
fn main() {}
 % cat c.rs
#[link_args = "test1234"]
extern {}
% rustc a.rs
c.rs:1:1: 1:26 warning: unused attribute, #[warn(unused_attributes)] on by default
c.rs:1 #[link_args = "test1234"]
       ^~~~~~~~~~~~~~~~~~~~~~~~~
note: in expansion of include!
a.rs:3:1: 3:17 note: expansion site
error: linking with `cc` failed: exit code: 1
note: [SNIP]
note: clang: error: no such file or directory: 'test1234'
```

**Comment by steveklabnik** [comment]
This seems to work now, instead saying that test1234 doesn't exist.
